lib/generators/solidus_stripe/install/templates/app/assets/stylesheets/spree/frontend/solidus_stripe.css

```diff
@@ -6,3 +6,8 @@ the installer will append this file to the app vendored assets here: 'vendor/ass
 .solidus-stripe-payment {
   padding: 1em 0;
 }
+
```

Comment:
Does `Dim disabled submit buttons` need a rewrite?

Reply:
Lol, I thought `Dim` was some sort of leftover vim shortcut that went wrong. 🤦, also because I know you don't use Vim!

Reply:
If I was to use vim you would see those weird extra characters all over the place! ⌨️ 🤣